rubin_scheduler/scheduler/surveys/surveys.py

```diff
@@ -196,6 +193,9 @@ def __init__(
         search_radius=None,
         alt_max=-9999,
         az_range=-9999,
+        target_name="",
+        observation_reason="FBS",
+        science_program="FBS",
```

Comment:
science_program is going to end up being the JSON BLOCK identifier, so it might be better if we don't set a default here .. although this is also probably something that is useful to discuss with Tiago. 
Leaving as "FBS" for now, when we're not actually using this, is probably also fine. Could replace both `science_program` and `observation_reason` with None for now as well.